### PR TITLE
Preparing 0.19.0.0 release

### PR DESCRIPTION
**Pull request by alanz**

### CODE REVIEW

**Comment by alanz** [comment]
@bubba Please let me know when the lsp-test stuff is in place, or if I can do this directly (once approved)

**Comment by cocreature** [comment]
Nice, thank you! The `ghcide` tests are all passing with this.

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Revision history for haskell-lsp
 
+## 0.19.0.0 -- 2019-12-14
+
+* Fix vfs line endings (@jneira)
+* Fix typo in .cabal (@turion)
+* Normalize file paths before converting to Uri's (@jneira)
+* Fixes to persistVirtualFile (@mpickering)
+* Fix response type for CodeLensResolve (@SquidDev)
```
Comment (SquidDev) [inline ChangeLog.md]: I think my changes got into the last release, just a little late in the day.

Comment (alanz) [reply ChangeLog.md]: Ok, I will check, I was looking at the PR history in github. And I just realised now we did not tag it.

**Pull request merged**